state/event_store.go

```diff
@@ -1,31 +1,33 @@
-package event
+package state
 
 import (
 	"fmt"
 	"sync"
+
+	"github.com/hashicorp/consul-terraform-sync/state/event"
 )
 
 const defaultEventCountLimit = 5
 
-// Store stores events
-type Store struct {
+// eventStore stores events
+type eventStore struct {
```

Comment:
It makes sense to me to make this private! Just wondering if the naming will be confusing because we've got the store interface and the in memory store implementation of that interface, but event store is kind of its own thing. 🤔  It probably won't be a big deal since it's private anyway, so not strongly advocating one way or another!

Reply:
That's a good point! Yea `eventStore` looks like it is an implementation of the `Store` interface.

Noting for posterity: we chatted about this offline and decided to go with `eventStorage` for now. This new name is still a bit close to `Store` but it will make it clear that `eventStorage` doesn't implement `Store` interface. Down the line, we can also readjust the `Store` name as well as we see how it gets consumed by other code. Thanks!